gulpfile.babel.js

```diff
@@ -0,0 +1,10 @@
+/* Sublime Text 3 Theme Builder */
+
+import gulp from 'gulp';
+
+// Use gulp-stats
+require('gulp-stats')(gulp);
+require('require-dir')('.gulp/tasks');
+
+// Set default script
+gulp.task('default', ['build']);
```

Comment:
I kinda like this because it's really explicit. Just wondering if can be more complex than this, cause if you remove the build file from tasks folder, this file shouldn't be aware of that (for a bit of decoupling).